configs/env-example

```diff
@@ -141,7 +141,7 @@ FILE_NAME= cypress.env.json
 FILE_PATH= SIMS\testing\functional\tests\cypress.env.json
 
 # Zeebe
-ZEEBE_ADDRESS=
+ZEEBE_ADDRESS=localhost:26500
```

Comment:
can you please explain, why this value is hardcoded here? is this intensional or for development purpose?

Reply:
Please keep in mind that this is the sample env. I added it here for developer reference to help to run it locally since the env file is used only for our local dev env and that hardcoded value is the one that will be created when the container is created in our local dev machine.

Reply:
I believe that is the local ZEBEE default port when u start camunda in local by make camunda